aea/cli/search.py

```diff
@@ -125,22 +125,21 @@ def _search_items(ctx, item_type_plural):
             "config_file": DEFAULT_SKILL_CONFIG_FILE,
         },
     }
-    if item_type_plural == "agents":
-        lookup_dir = registry
-    else:
-        lookup_dir = AEA_DIR
+    if item_type_plural != "agents":
+        # look in aea distribution for default packages
         _get_details_from_dir(
```

Comment:
this part should have been applied for all packages and agents, now it is. Local lookup should only happen for the packages, now it is.